server/plugin/webhook.go

```diff
@@ -270,6 +272,10 @@ func (p *Plugin) postPullRequestEvent(event *github.PullRequestEvent) {
 		}
 
 		if action == "opened" {
+			policy := bluemonday.StrictPolicy()
+			policy.SkipElementsContent("details")
+			newPRMessage = strings.ReplaceAll(newPRMessage, "<details>", "details collapsed.<details>")
+			newPRMessage = policy.Sanitize(newPRMessage)
```

Comment:
Great solution! I have one suggestion to change the substitution string

```suggestion
			newPRMessage = strings.ReplaceAll(newPRMessage, "<details>", "`(details collapsed)`<details>")
			newPRMessage = policy.Sanitize(newPRMessage)
```

Reply:
Done